src/react-native/react-native-wizard.ts

```diff
@@ -0,0 +1,256 @@
+    selectedProject,
+  );
+
+  if (confirmedFirstException) {
+    clack.outro(
+      `${chalk.green('Everything is set up!')}
+
+   ${chalk.dim(
+     'If you encounter any issues, let us know here: https://github.com/getsentry/sentry-react-native/issues',
+   )}`,
+    );
+  } else {
+    clack.outro(
+      `${chalk.dim(
+        'Let us know here: https://github.com/getsentry/sentry-react-native/issues',
+      )}`,
+    );
+  }
+}
+
+function addSentryInit({ dsn }: { dsn: string }) {
+  const prefixGlob = '{.,./src}';
+  const suffixGlob = '@(j|t|cj|mj)s?(x)';
+  const universalGlob = `App.${suffixGlob}`;
+  const jsFileGlob = `${prefixGlob}/+(${universalGlob})`;
+  const jsPath = getFirstMatchedPath(jsFileGlob);
+  if (!jsPath) {
+    clack.log.warn(
+      `Could not find main App file using ${chalk.bold(jsFileGlob)}.`,
+    );
+    return;
+  }
+  const jsRelativePath = path.relative(process.cwd(), jsPath);
+
+  const js = fs.readFileSync(jsPath, 'utf-8');
+  if (
+    doesJsCodeIncludeSdkSentryImport(js, { sdkPackageName: RN_SDK_PACKAGE })
+  ) {
+    clack.log.warn(`${chalk.bold(jsRelativePath)} already includes Sentry.`);
+    return;
+  }
+
+  const newContent = addSentryInitWithSdkImport(js, { dsn });
+
+  clack.log.success(
+    chalk.greenBright(
+      `Added ${chalk.bold('Sentry.init')} to ${chalk.bold(jsRelativePath)}.`,
+    ),
+  );
+
+  fs.writeFileSync(jsPath, newContent, 'utf-8');
+  clack.log.success(
+    chalk.greenBright(`${chalk.bold(jsRelativePath)} changes saved.`),
+  );
+}
+
+async function confirmFirstSentryException(project: SentryProjectData) {
+  const projectsIssuesUrl = `${project.organization.links.organizationUrl}/issues/?project=${project.id}`;
+
+  clack.note(`To make sure everything is set up correctly, put the following code snippet into your application.
+The snippet will create a button that, when tapped, sends a test event to Sentry.
+
+After that check your project issues:
+
+${chalk.cyan(projectsIssuesUrl)}
+
+<Button title='Try!' onPress={ () => { Sentry.captureException(new Error('First error')) }}/>`);
+
+  const firstErrorConfirmed = clack.confirm({
+    message: `Have you successfully sent a test event?`,
+  });
+
+  return firstErrorConfirmed;
+}
+
+async function patchXcodeFiles({ authToken }: { authToken: string }) {
+  await addSentryCliConfig(authToken, {
+    ...propertiesCliSetupConfig,
+    name: 'source maps and iOS debug files',
+    filename: 'ios/sentry.properties',
+  });
+
+  if (platform() === 'darwin') {
+    await podInstall('ios');
+  }
+
+  const xcodeProjectPath = getFirstMatchedPath(XCODE_PROJECT);
+  if (!xcodeProjectPath) {
+    clack.log.warn(
+      `Could not find Xcode project file using ${chalk.bold(XCODE_PROJECT)}.`,
+    );
+    return;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+  const xcodeProject = xcode.project(xcodeProjectPath);
```

Comment:
l: just wondering if there's a @types/xcode package or some other way of getting types for `xcode` so that we can get rid of the eslint ignores?

Reply:
Sadly there is none -> https://github.com/apache/cordova-node-xcode/issues/64, but I think for the few functions we could add the types ourselves. Maybe next PR.